includes/class-zoninator-api-controller.php

```diff
@@ -373,95 +412,135 @@ private function _get_param( $object, $var, $default = '', $sanitize_callback =
 		return $value;
 	}
 
+	/**
+	 * Params for Create Zone
+	 *
+	 * @return array[]
+	 */
 	public function _params_for_create_zone() {
 		return array(
-			'name' => array(
-				'type' => 'string',
+			'name'        => array(
+				'type'              => 'string',
 				'sanitize_callback' => array( $this, 'sanitize_string' ),
-				'default' => '',
-				'required' => false
+				'default'           => '',
+				'required'          => false,
 			),
-			'slug' => array(
-				'type' => 'string',
+			'slug'        => array(
+				'type'              => 'string',
 				'sanitize_callback' => array( $this, 'sanitize_string' ),
-				'default' => '',
-				'required' => false,
+				'default'           => '',
+				'required'          => false,
 			),
 			'description' => array(
-				'type' => 'string',
+				'type'              => 'string',
 				'sanitize_callback' => array( $this, 'sanitize_string' ),
-				'default' => '',
-				'required' => false,
-			)
+				'default'           => '',
+				'required'          => false,
+			),
 		);
 	}
 
+	/**
+	 * Params for Update Zone
+	 *
+	 * @return array[]
+	 */
 	public function _params_for_update_zone() {
 		return array(
-			'name' => array(
-				'type' => 'string',
+			'name'        => array(
+				'type'              => 'string',
 				'sanitize_callback' => array( $this, 'sanitize_string' ),
-				'required' => false
+				'required'          => false,
 			),
-			'slug' => array(
-				'type' => 'string',
+			'slug'        => array(
+				'type'              => 'string',
 				'sanitize_callback' => array( $this, 'sanitize_string' ),
-				'required' => false,
+				'required'          => false,
 			),
 			'description' => array(
-				'type' => 'string',
+				'type'              => 'string',
 				'sanitize_callback' => array( $this, 'sanitize_string' ),
-				'required' => false,
-			)
+				'required'          => false,
+			),
 		);
 	}
 
+	/**
+	 * Get zone ID params
+	 *
+	 * @return array[]
+	 */
 	public function _get_zone_id_param() {
 		return array(
 			'zone_id' => array(
 				'type'              => 'integer',
 				'validate_callback' => array( $this, 'is_numeric' ),
 				'sanitize_callback' => 'absint',
-				'required'          => true
-			)
+				'required'          => true,
+			),
 		);
 	}
 
+	/**
+	 * Get zone post rest route params
+	 *
+	 * @return array[]|bool[]|string[][]|Zoninator_Api_Controller[]
```

Comment:
Due to the complexity of nested mixed types, then:
```suggestion
	 * @return array Params.
```
would be sufficient here. The other entries are not correct, and would be generally used for index arrays, not associative arrays.